META.json

```diff
@@ -0,0 +1,37 @@
+{
+   "name": "wltree",
+   "abstract": "This is adevens patched version of Postgres ltree module",
+   "version": "2.0",
+   "license": {
+      "PostgreSQL": "http://www.postgresql.org/about/licence"
+   },
+   "provides": { 
+      "wltree": {
+         "file": "sql/pg_lock_pool.sql",
+         "version": "0.2.0",
+         "abstract": "This is adevens patched version of Postgres ltree module"
+      }
+   },
+   "meta-spec": {
+     "version": "2.0",
+     "url": "http://pgxn.org/meta/spec.txt"
+   },
+   "description": "This is adevens patched version of Postgres ltree module",
+   "prereqs": {
+      "runtime": {
+        "requires": {
+           "PostgreSQL": "9.3.0"
```

Comment:
Since we test only PostgreSQL 10+ it might make sense to change it to `"10.0"`:
https://github.com/adjust/wltree/blob/master/.github/workflows/main.yml#L14

Reply:
I agree, no need to test very old versions.